components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultAuthenticationRequestHandler.java

```diff
@@ -919,4 +922,24 @@ private AuthenticationResult getAuthenticationResult(HttpServletRequest request,
         return authenticationResult;
     }
 
+    private void setAuthenticatedIDPByApps(SessionContext sessionContext, String applicationName) {
+
+        if (log.isDebugEnabled()) {
+            log.debug("Getting shared authenticatedIdpData from session context and setting it for"
+                    + "application: " + applicationName);
```

Comment:
When getting printed this will show a lack of space as 'forapplication'

Reply:
fixed